scipy/stats/stats.py

```diff
@@ -5939,3 +5942,206 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def rvs_ratio_uniforms(pdf, size=1, x1=None, x2=None, y2=None, shift=0,
+                       maxiter=None, bounds=None, random_state=None):
+    """
+    Generate random samples from a probability density function using the
+    ratio-of-uniforms method.
+
+    Parameters
+    ----------
+    pdf : callable
+        The probability density function of the distribution.
+    size : int or tuple of ints, optional
+        Defining number of random variates (default is 1).
+    x1 : float, optional
+        The lower bound of the bounding rectangle in the x-direction.
+        Default is None. In that case, attempt to find bound numerically.
+    x2 : float, optional
+        The upper bound of the bounding rectangle in the x-direction.
+        Default is None. In that case, attempt to find bound numerically.
+    y2 : float, optional
+        The upper bound of the bounding rectangle in the y-direction.
+        Default is None. In that case, attempt to find bound numerically.
+    shift : float, optional.
```

Comment:
Is the name `shift` open to being renamed? Would you consider just using `c`?  Reasoning behind the suggestion:
- `c` matches what is used in the `Notes` section
- `c` doesn't have any other meaning in this context which might cause confusion, whereas `shift` has multiple  potential meanings.
- `c` is shorter

Reply:
Yes, i am open to renaming it and I agree that `shift` is not self-explanatory, so why not just call it `c` as in the notes.